addons/website_event/views/event_templates_list.xml

```diff
@@ -261,17 +266,28 @@
                         </li>
                     </ul>
                 </div>
-            </div>    
+            </div>
         </div>
     </xpath>
 </template>
 
+<!-- Grid or List Display Selection -->
+<template id="website_event.add_grid_or_list_option" active="True" name="Grid or List button">
+    <t t-set="_activeClasses" t-translation="off">active</t>
+    <div t-attf-class="o_wevent_apply_layout btn-group d-flex" t-att-data-active-classes="_activeClasses">
+        <input type="radio" class="btn-check" name="wevent_events_layout" id="o_wevent_apply_grid"  t-att-checked="'checked' if layout_mode == 'grid' else None" value="grid"/>
+        <label t-attf-class="btn btn-{{navClass}} #{_activeClasses if layout_mode == 'grid' else None} o_wevent_apply_grid" title="Grid" for="o_wevent_apply_grid">
+            <i class="fa fa-th-large"/>
+        </label>
+        <input type="radio" class="btn-check" name="wevent_events_layout" id="o_wevent_apply_list" t-att-checked="'checked' if layout_mode == 'list' else None" value="list"/>
+        <label t-attf-class="btn btn-{{navClass}} #{_activeClasses if layout_mode == 'list' else None} o_wevent_apply_list" title="List" for="o_wevent_apply_list">
+            <i class="oi oi-view-list"/>
+        </label>
+    </div>
```

Comment:
Again, no idea why we need navClass and _activeClasses, could be removed if no good reason to keep them. Please be careful when importing code from elsewhere, and recheck variable used. Those must be justifiable or simplified :)

We can probably simplify and directly toggle 'active' in the js for example and use it here in the template.
navClass is used in a lot of places in eCommerce, not sure it has sense here.